<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" ignored="1" total="3" passed="1">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="7724" started-at="2021-02-28T23:25:15 IST" finished-at="2021-02-28T23:25:22 IST">
    <groups>
    </groups>
    <test name="Default test" duration-ms="7724" started-at="2021-02-28T23:25:15 IST" finished-at="2021-02-28T23:25:22 IST">
      <class name="saucelabs.LogintoApplication">
        <test-method status="PASS" signature="logintosauce(java.lang.String, java.lang.String)[pri:0, instance:saucelabs.LogintoApplication@27a8c74e]" name="logintosauce" duration-ms="6514" started-at="2021-02-28T23:25:15 IST" data-provider="getusernamepassword" finished-at="2021-02-28T23:25:21 IST">
          <params>
            <param index="0">
              <value>
                <![CDATA[standard_user]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[secret_sauce]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- logintosauce -->
        <test-method status="FAIL" signature="verifylogin()[pri:0, instance:saucelabs.LogintoApplication@27a8c74e]" name="verifylogin" duration-ms="95" started-at="2021-02-28T23:25:21 IST" finished-at="2021-02-28T23:25:21 IST">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [Prducts] but found [Products]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [Prducts] but found [Products]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:629)
at org.testng.Assert.assertEquals(Assert.java:639)
at saucelabs.LogintoApplication.verifylogin(LogintoApplication.java:53)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:132)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:599)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:174)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:822)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:147)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(ArrayList.java:1257)
at org.testng.TestRunner.privateRun(TestRunner.java:764)
at org.testng.TestRunner.run(TestRunner.java:585)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1218)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
at org.testng.TestNG.runSuites(TestNG.java:1069)
at org.testng.TestNG.run(TestNG.java:1037)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifylogin -->
        <test-method status="PASS" signature="teardown()[pri:0, instance:saucelabs.LogintoApplication@27a8c74e]" name="teardown" is-config="true" duration-ms="1042" started-at="2021-02-28T23:25:21 IST" finished-at="2021-02-28T23:25:22 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- teardown -->
      </class> <!-- saucelabs.LogintoApplication -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
